scripts/arm32_ci_script.sh

```diff
@@ -178,6 +178,55 @@ function cross_build_corefx {
     ROOTFS_DIR="$__ARMRootfsMountPath" CPLUS_INCLUDE_PATH=$LINUX_ARM_INCPATH CXXFLAGS=$LINUX_ARM_CXXFLAGS ./build-managed.sh -$__buildConfig -skipTests -BuildPackages=false
 }
 
+# Cross builds corefx using Docker image
+function cross_build_corefx_with_docker {
+    # Cross building using Docker image
+    currentWorkingDirectory=`pwd`
+
+    # Build rootfs in docker
+    #dockerCmd="sudo docker run -i --rm -v $currentWorkingDirectory:/opt/corefx -w /opt/corefx chcosta/dotnetcore:ubuntu1404_cross_prereqs_v1"
+    #dockerCmd="sudo docker run -i --rm -v $currentWorkingDirectory:/opt/corefx -w /opt/corefx t2wish/dotnetcore:ubuntu1404_cross_prereqs_v2"
+    # Build rootfs in Docker
+    #if [ "$__buildArch" == "arm" ]; then
+    #    buildRootfsCmd="/opt/corefx/cross/build-rootfs.sh arm"
+    #elif [ "$__buildArch" == "armel" ]; then
+    #    buildRootfsCmd="/opt/corefx/cross/build-rootfs.sh armel tizen"
+    #else
+    #    echo "ERROR: unknown buildArch $__buildArch"
+    #    exit 1
+    #fi
+    #$dockerCmd $buildRootfsCmd
+
+    # Build rootfs in host
+    if [ "$__buildArch" == "arm" ]; then
+        sudo apt-get update
+        sudo apt-get -y install  qemu qemu-user-static binfmt-support debootstrap
+        buildRootfsCmd="sudo ./cross/build-rootfs.sh arm xenial --skipunmount"
```

Comment:
This was a draft PR to see whether it's working with existing netci.groovy.
I will add a logic to select trusty (14.04) and xenial (16.04), so we can later choose proper version from netci.groovy.